src/pages/integrations/figma.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { LandingPage } from '../../component/landingPages/LandingPage';
+import { LandingPageHeadline } from '../../component/landingPages/LandingPageHeadline';
+import { LandingPageDescription } from '../../component/landingPages/LandingPageDescription';
+import { LandingPageInstallationStep } from '../../component/landingPages/LandingPageInstallationStep';
+import { LandingPageInstallation } from '../../component/landingPages/LandingPageInstallation';
+import FigmaLogo from '../../../static/img/technologies/logo-figma.svg';
+import { LandingPageActions } from '../../component/landingPages/LandingPageActions';
+import { MoreIntegrations } from '../../component/landingPages/LandingPageMoreIntegrations';
+import Head from '@docusaurus/Head';
+
+const DocsLinks = ({ primary }: { primary?: boolean }) => (
+  <LandingPageActions
+    docs={{ link: '/platform/figma-plugin/usage' }}
+    githubRepo={{
+      explicitLink: true,
+      link: 'https://github.com/tolgee/figma-plugin',
+    }}
+    primary={primary}
+  />
+);
+
+export default function Figma() {
+  return (
+    <LandingPage title={'Tolgee Figma plugin'}>
+      <Head>
+        <meta
+          name="description"
+          content="Connect your Figma designs with Tolgee. This integration will make the process of localizing your designs faster, easier, and more efficient."
```

Comment:
meta description should be a bit longer. e.g. you can add: Connect your Figma designs with Tolgee localization platform